src/components/Request/Request.jsx

```diff
@@ -1,2 +1,41 @@
 import React from 'react';
-impor
\ No newline at end of file
+import { useResty } from '../../components/Hooks/RestyProvider';
+
+const Request = () => {
+  const { url, method, body, onChange, onSubmit } = useResty();
+
+  return (
+    <form onSubmit={onSubmit}>
+      <input type="text" name="url" value={url} onChange={onChange} />
+  
+      <label>
+        <input type="radio" name="method" value="POST" checked={method === 'POST'} onChange={onChange} />
+        POST
+      </label>
+  
+      <label>
+        <input type="radio" name="method" value="GET" checked={method === 'GET'} onChange={onChange} />
+        GET
+      </label>
+  
+      <label>
+        <input type="radio" name="method" value="PUT" checked={method === 'PUT'} onChange={onChange} />
+        PUT
+      </label>
+  
+      <label>
+        <input type="radio" name="method" value="PATCH" checked={method === 'PATCH'} onChange={onChange} />
+        PATCH
+      </label>
+  
+      <label>
+        <input type="radio" name="method" value="DELETE" checked={method === 'DELETE'} onChange={onChange} />
+        DELETE
+      </label>
+  
+      <textarea name="body" value={body} onChange={onChange}></textarea>
+      <button>Request</button>
+    </form>
+  );
+};
+
```

Comment:
something feels missing on this file causing an error, let me know if you need help finding it! Hint: the rest of your files aren't receiving this componenet